Classes/Wizard/StateCreateBeGroup.php

```diff
@@ -66,11 +66,11 @@ protected function createBeGroup(BaseDto $siteData): int
 
         // Set common mountpoint
         if ($siteData->getCommonMountPointUid()) {
-            $data['be_groups'][$newUniqueId]['file_mountpoints'] = $siteData->getCommonMountPointUid();
+            $data['be_groups'][$newUniqueId] = ['file_mountpoints' => $siteData->getCommonMountPointUid()];
         }
 
         // Set created mountpoint
-        if ($siteData->getMountId()) {
+        if (isset($data['be_groups'][$newUniqueId]['file_mountpoints']) && $siteData->getMountId()) {
```

Comment:
This does not cover all the cases: getMountId() should be set event if $data['be_groups'][$newUniqueId]['file_mountpoints'] is not set.

Reply:
Okay, I unterstand. Of course, then it makes no sense.